### ISSUE TITLE
problem/feature with repeat point action

### ISSUE THREAD

**Issue by johbuch** [mautic/mautic#7320]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
When you create a point action for opening an email, you can choose to repeat the action. That means if the contact opens different emails, he will be scored.
The thing is, he will also be scored if he opens multiple times the same email. So he will get more point than he really should have.

It is nice to have the possibility to repeat the action but it could be better if we could limit to add points for the 1st email.
| Q   | A
| --- | ---
| Mautic version | 2.15
| PHP version | 
| Browser | 

### Steps to reproduce
1. create a point action, choose opens an email. Enable to repeat the action.
2. Send an email to a contact.
3. Open the email.
4. Check that your contact got X points.
5. Open again the email. See that you got points again.

It is only about **opening emails**
 
### Log errors

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by samjaninf** [comment]
Honestly... this doesn't sound like a bug. If you want them to only get points for opening the email once, then you should either make the action apply only once, or add the edit contact points to your campaign. If you need them to get points only once from each email and you aren't using a campaign then I would just add a separate action for those emails.

Or am I missing something? :(

**Comment by RCheesley** [comment]
@johbuch would be great if you could test the PR #7330 which should address this issue.
